content/general/concepts/cms/cms.md

```diff
@@ -0,0 +1,17 @@
+​A content management system (CMS) is a software application that allows users to create, edit, and manage digital content such as text, images, videos, and other multimedia files. A CMS typically provides a user-friendly interface that allows users to easily add, delete, and modify content without the need for technical knowledge or coding skills. The primary purpose of a CMS is to make it easier for organizations to manage and maintain their online presence by providing a central platform for organizing and publishing content.
```

Comment:
We're missing a metadata section

Reply:
@SSwiniarski how do I get the metadata?

Reply:
Look at the metadata section in the content standards [here](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) to see what goes in the metadata, then look at the source of the entry template [here](https://github.com/Codecademy/docs/blob/main/documentation/entry-template.md) to see how to format it in the file.

Reply:
You can also look at the source of a similar General entry to get an idea what to add.

Reply:
Done!